backend/src/serverless/integrations/workers/devtoWorker.ts

```diff
@@ -30,6 +35,20 @@ async function devtoWorker(body: DevtoIntegrationMessage) {
     // Inject user and tenant to IRepositoryOptions
     const userContext = await getUserContext(tenant)
 
+    const memberAttributesService = new MemberAttributeSettingsService(userContext)
+
+    await memberAttributesService.createPredefined(DevtoMemberAttributes)
+
+    await memberAttributesService.createPredefined(
+      TwitterMemberAttributes.filter((a) => a.name === MemberAttributes.URL.name),
```

Comment:
Maybe create a helper method to just do `pickAttributes(names, arrayOfAttributes)` and call it like this (where `MemberAttributes` is an actual enum as well): `pickAttributes([MemberAttributes.URL], TwitterMemberAttributes)`.

Reply:
it'll be very uncommon case (most probably never happened before) so I'm not sure how to handle it, because there'll most likely be data already populated in member.attributes with these fields

Reply:
helper function included